packages/terra-table/src/subcomponents/ColumnHeader.jsx

```diff
@@ -19,9 +19,21 @@ const propTypes = {
   headerHeight: PropTypes.string.isRequired,
 
   /**
-   * Number that specifies the height of the table in pixels.
-   */
+  * Number that specifies the height of the data grid in pixels.
```

Comment:
This is the Terra Table component.  I would not expect the prop to reference the data grid.